src/features/lightspeed/inlineSuggestions.ts

```diff
@@ -876,9 +876,11 @@ export async function inlineSuggestionHideHandler(userAction?: UserAction) {
   switch (action) {
     case UserAction.REJECTED: {
       console.log("[inline-suggestions] User rejected the inline suggestion.");
+      break;
```

Comment:
Good catch!

Reply:
Need to send kudos to eslint folks, not me. There are also lots of other changes that we will need to make, but we can do them later, i marked them as warnings only, so we will be nagged on PRs about them, at least until we can mark them as errors.